test/unit/driver-specs.js

```diff
@@ -119,7 +119,8 @@ describe('driver.js', () => {
         }).should.throw;
       });
       describe('nativeWebScreenshot', function () {
-        let listLength;
+        let list;
+        let nativeWebScreenshotFilter = item => { return item[0] === "GET" && item[1].test('/session/xxx/screenshot/');};
```

Comment:
change to 
`item => item[0] === 'GET' && item[1].test('/session/xxx/screenshot')`